app/views/repair_requests/new.html.erb

```diff
@@ -59,7 +59,7 @@
       <%= form.form_group :description do %>
         <%= form.label :description, "Problem description" %>
         <%= form.error_messages :description %>
-        <%= form.text_area :description, class: 'govuk-textarea govuk-!-margin-bottom-1', maxlength: 250, rows: 4, onkeyup: 'characterCount()' %>
+        <%= form.text_area :description, value: @problem_description, class: 'govuk-textarea govuk-!-margin-bottom-1', maxlength: 250, rows: 4, onkeyup: 'characterCount()' %>
```

Comment:
We shouldn't set the `value: ` attribute on the input, but instead set the property on the model, which is responsible for holding state, and then letting the form helpers pick it up automatically.

Reply:
Ah I now see how this works. Rails is so clever!

Reply:
:railway_track: